contents/docs/configuring-posthog/scaling-posthog.md

```diff
@@ -4,7 +4,7 @@ sidebar: Docs
 showTitle: true
 ---
 
-A standard instance of PostHog using PostgreSQL is pretty efficient for storing large volumes of data. We've seen volumes of **5 million events/day** and hundreds of requests a second on Heroku's cheapest tier with no problems.
+A standard instance of PostHog using PostgreSQL is pretty efficient for storing large volumes of data. We've seen volumes of **1 million events/day** and hundreds of requests a second on Heroku's standard tier with no problems.
```

Comment:
Let's make this a lot more pessimistic, something like: "Ingestion of a lot of events should not be a problem. However doing any form of querying (especially over larger periods of time) will slow down massively as Postgres is not optimised for analytics payloads. If you are experiencing issues we recommend scaling up your postgres database a lot, or upgrading to using Clickhouse"

maybe better worded